src/wifi.rs

```diff
@@ -1625,6 +1664,8 @@ fn matches_wifi_event(event: &WifiEvent) -> bool {
     )
 }
 
+/// Wraps a [`WifiDriver`] or [`EspWifi`], and offers strictly synchronous (blocking)
+/// function calls for their functionality.
```

Comment:
You might want to add an example here in that wrapping is possible both by moving the `WifiDriver` / `EspWifi` instance into `BlockingWifi`, as well as by mutable borrowing it (i.e. `&mut wifi_driver`), where the latter allows one to re-use the driver elsewhere once `BlockingWifi` is dropped.

Reply:
Alas, I wasn't even aware that `BlockingWifi` was wrapping a `EspWifi` instance until I read the source code - I don't have a use case where I'd reuse the driver by dropping `BlockingWifi`, much less I feel confident to write a reliable example.

I'd rather leave this for a future improvement.